src/class-fields/computed-name-empty-var-empty-function.case

```diff
@@ -0,0 +1,29 @@
+// Copyright (C) 2016 the V8 project authors, 2017 Igalia S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Computed name var not initialized, empty function
+info: |
+  1.1 New Productions
+
+  [...]
+
+  FieldDefinitionList [Yield, Await]:
+    FieldDefinition [?Yield, ?Await]
+    FieldDefinitionList  [?Yield, ?Await], FieldDefinition [?Yield, ?Await]
+template: default
+includes: [propertyHelper.js]
+---*/
+
+//- body
+  ['a']; b(){}
+
+//- assertions
+  assert.sameValue(this.a, undefined);
+  verifyEnumerable(this, "a");
+  verifyWritable(this, "a");
+  verifyConfigurable(this, "a");
+
+  assert.sameValue(typeof Object.getPrototypeOf(this).b, "function");
+  verifyNotEnumerable(Object.getPrototypeOf(this), "b");
+  verifyConfigurable(Object.getPrototypeOf(this), "b");
```

Comment:
This is the generated test: 

```js
class Base {}
class C extends Base {
  ['a']; b(){}

  constructor() {
    super();
    assert.sameValue(this.a, undefined);
      verifyEnumerable(this, "a");
      verifyWritable(this, "a");
      verifyConfigurable(this, "a");

      assert.sameValue(typeof Object.getPrototypeOf(this).b, "function");
      verifyNotEnumerable(Object.getPrototypeOf(this), "b");
      verifyConfigurable(Object.getPrototypeOf(this), "b");
  }
}

const c = new C();
```


I want it to read: 

```js
class Base {}
class C extends Base {
  ['a']; b(){}

  constructor() {
    super();
    assert.sameValue(this.a, undefined);
      verifyEnumerable(this, "a");
      verifyWritable(this, "a");
      verifyConfigurable(this, "a");

      assert.sameValue(typeof this.b, "function");
      verifyNotEnumerable(this, "b");
      verifyConfigurable(this, "b");
  }
}

const c = new C();
```

Because, as a follow up, I want to update the cases and templates with additions necessary to produce this: 

```js
class Base {}
class C extends Base {
  ['a']; b(){}

  constructor() {
    super();
    assert.sameValue(this.a, undefined);
    verifyEnumerable(this, "a");
    verifyWritable(this, "a");
    verifyConfigurable(this, "a");

    assert.sameValue(typeof this.b, "function");
    verifyNotEnumerable(this, "b");
    verifyConfigurable(this, "b");
  }
}

assert.sameValue(C.prototype.a, undefined);
assert.sameValue(typeof C.prototype.b, "function");

const c = new C();
```

(Or something similar to that.